app/views/admin/orders/index.html.erb

```diff
@@ -18,6 +19,9 @@
         <td><%= order.reward_snapshot.description %></td>
         <td><%= order.reward_snapshot.price.to_i %></td>
         <td><%= order.created_at.strftime("%d-%m-%y %H:%M") %></td>
+        <td><%if order.status == 'not_delivered'%>
```

Comment:
Let's use enum's helper method here - as far as I remember it should be something like:

```suggestion
        <td><% if order.not_delivered? %>
```

Reply:
You are right i changed it